### PR TITLE
Include the step name in finishRunning() errors

### PR DESCRIPTION
**Pull request by fejta**
Hopefully this helps create better clusters in http://go.k8s.io/triage
/assign @rmmh @krzyzacy

### CODE REVIEW

File: kubetest/util.go [hunk 1]
```diff
@@ -137,7 +137,10 @@ func finishRunning(cmd *exec.Cmd) error {
 				cmd.Process.Kill()
 			}
 		case err := <-finished:
-			return err
+			if err != nil {
+				return fmt.Errorf("error during %s: %v", stepName, err)
+			}
+			return nil
```
Comment (krzyzacy) [inline kubetest/util.go]: return err?

Comment (fejta) [inline kubetest/util.go]: Done

Comment (krzyzacy) [inline kubetest/util.go]: oh it was a return :-( nvm

**Pull request merged**